python/MetrologyData.py

```diff
@@ -268,11 +269,12 @@ def plot_statistics(self, nsigma=5, title=None, zoffset=0):
                                nsigma), (0.05, 0.8), xycoords='axes fraction')
 
         # Overlay a Gaussian with the same sigma and correct normalization
-        x = np.arange(np.min(dz), np.max(dz), 0.1)
```

Comment:
You're right, of course.  I was hasty, and the example data set that I ran had self.sensor.stdev_filt = 4.07, which was close to rounded.  I've removed lim now, and set bins=bins, i.e., the value of the keyword argument.